### DISCUSSION TITLE
Deployments stuck in Queued

### DISCUSSION THREAD

**Discussion by suCozy** [vercel/community#3103] [Help] [answered]
### Summary

So, I was trying to deploy my project into Vercel with github and its stuck now in that Queue thing I dont know what to do, like do I just wait till it works?

url: [website-front-production-pkrk1j1na-vetplace.vercel.app](https://website-front-production-pkrk1j1na-vetplace.vercel.app/)

help plz....

### Steps to Reproduce

So, I was trying to deploy my project into Vercel with github and its stuck now in that Queue thing I dont know what to do, like do I just wait till it works?

**Comment by amyegan** [comment, marked as answer]
Thanks for reporting this. It looks like things are back to normal now. Please let me know if you continue to see problems 🙏 

https://www.vercel-status.com/incidents/qvwqfrr2kp3l

**Reply by toniengelhardt** [reply]
I have the same issue since 2 days, all builds are stuck 😭
